inc/namespace.php

```diff
@@ -9,9 +9,13 @@
 
 const DISABLE_ACTION = 'hm_disableaccounts_disable';
 const DISABLE_ACTION_BULK = 'hm_disableaccounts_disable_bulk';
+const DISABLE_USER_ACTION = 'hm_disableaccounts_disable_user';
+const DISABLED_USER_ACTION = 'hm_disableaccounts_disabled_user';
 const ENABLE_ACTION = 'hm_disableaccounts_enable';
 const ENABLE_ACTION_BULK = 'hm_disableaccounts_enable_bulk';
 const DISABLED_META_KEY = '_hm_disableaccounts_disabled';
+const REENABLE_USER_ACTION = 'hm_disableaccounts_reenable_user';
+const REENABLED_USER_ACTION = 'hm_disableaccounts_reenabled_user';
```

Comment:
These are only used once, so very low possibility of making a mistake - I think they should be inlined instead.

Reply:
I can do that, on our project we import some of these constants to avoid mistakes so I thought it best to continue the pattern

Reply:
imo it’s less readable this way, since you have a bit of overabstraction.